Aspose.Slides.WebExtensions/TemplateCompilationException.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) 2001-2020 Aspose Pty Ltd. All Rights Reserved.
+
+using System.Collections.Generic;
+using System;
+
+namespace Aspose.Slides.WebExtensions
+{
+    class TemplateCompilationException : Exception
```

Comment:
@polonair 

Where is this exception thrown? I see catch, but no throw for it.

Reply:
Thank you, I will recheck and fix it